index.html

```diff
@@ -2122,17 +2139,17 @@ <h3>Media Types</h3>
                 <p>
                     <span class="rfc2119-assertion" id="arch-media-type">All
                         data (a.k.a. content) exchanged when activating Interaction Affordances MUST be identified by a Media Type
-                        [[!RFC6838]] within the Protocol Binding.</span>
-                    Media Types are IANA-managed labels to identify representation formats, for instance
+                        [[!RFC6838]] in the Protocol Binding.</span>
+                    Media Types are labels to identify representation formats, for instance
                     <code>application/json</code>
                     for JSON [[!RFC8259]] or
                     <code>application/cbor</code>
-                    for CBOR [[!RFC7049]].
+                    for CBOR [[!RFC7049]]. They are managed by IANA. 
                 </p>
                 <p>
                     Some Media Types might need additional parameters to fully specify the representation format used.
                     Examples are <code>text/plain; charset=utf-8</code> or <code>application/ld+json; profile="http://www.w3.org/ns/json-ld#compacted"</code>.
-                    This needs to be taken into account in particular when describing data to be sent to <a>Things</a>.
+                    This needs to be taken into account in particular when describing data to be sent to, or received by <a>Things</a>.
```

Comment:
Please remove "received by", because in that case the Thing must apply and set the content type in the message. A media Type hint would be enough.
When sending to Things, the Consumer must know the exact content type to formulate a correct request. A media type hint would not be enough. Hence "in particular"  for sending to Things.

Reply:
ok